examples/test/VisualizationByIdentifier_test.js

```diff
@@ -110,3 +114,14 @@ test("Measure value filter should render", async t => {
         .expect(chartValues.nth(1).textContent)
         .eql("57,264,327.95");
 });
+
+test("Measure value filter with treat null value as zero should render", async t => {
+    const pivotTableSelector = Selector("#measure-value-filter-treat-null-as-zero ~ div .s-pivot-table");
+
+    await waitForPivotTableStopLoading(t, pivotTableSelector);
+    // check applied filter
```

Comment:
Please read my comment for similar issue above (blank line below the comment line).